pkg/cmdutils/config.go

```diff
@@ -266,6 +246,14 @@ func loadFromEnv() *ClusterConfig {
 			config.AuthPlugin = props.GetString("authPlugin", "")
 			config.TLSEnableHostnameVerification = props.GetBool("tlsEnableHostnameVerification", false)
 		}
+	} else {
+		ctxConf, err := readConfigFile()
```

Comment:
This is a logic change, don't load the current context if the `PULSAR_CLIENT_CONF` variable was specified.  In other words, don't try to combine the info from two config files.